charts/amazon-cloudwatch-observability/templates/_helpers.tpl

```diff
@@ -17,8 +17,8 @@ Helper function to modify cloudwatch-agent config
 {{- $agent := set $configCopy "agent" $agentRegion }}
 {{- end }}
 
-{{- $appSignals := pluck "app_signals" $configCopy.logs.metrics_collected | first }}
-{{- if and (hasKey $configCopy.logs.metrics_collected "app_signals") (empty $appSignals.hosted_in) }}
+{{- $appSignals := pluck "application_signals" $configCopy.logs.metrics_collected | first }}
```

Comment:
Should we be supporting both names ? are we deprecating app_signals altogether ?

Reply:
Since helm is a GA feature, I don't think we need to be compatible with `app_signals`. It is deprecated. We will always recommend using `application_signals` in any case.